intercom_new_messages_block.py

```diff
@@ -50,21 +55,23 @@ def __init__(self):
     def configure(self, context):
         super().configure(context)
         self._create_web_server()
-        response = self._request('post', body={
-            "service_type": "web",
-            "topics": ["conversation.user.created"],
-            "url": self.callback_url(),
-        })
-        if response.status_code != 200:
-            raise Exception
-        self._subscription_id = response.json()["id"]
+        if self.manage_webhook():
```

Comment:
Just to confirm, this work-around for waiting on a token is still wanted?  Its not just wanted/needed in the `intercom_event_subscriptions_block`?

Reply:
Good question, @hansmosh want to weight in?

Reply:
I think we always want the managed webhook for the new messages block. My understanding is that you wouldn't want the managed webhook because you don't have the special access token yet, but you need the special access token for new messages anyway.

Reply:
It looks like anyone using this would want to get an expanded scope token anyway, so I'm going to agree and remove the `manage_webhook` option